src/eynollah/eynollah.py

```diff
@@ -1023,136 +909,6 @@ def do_prediction(
         gc.collect()
         return prediction_true
 
-    def do_padding_with_scale(self, img, scale):
-        h_n = int(img.shape[0]*scale)
-        w_n = int(img.shape[1]*scale)
-
-        channel0_avg = int( np.mean(img[:,:,0]) )
-        channel1_avg = int( np.mean(img[:,:,1]) )
-        channel2_avg = int( np.mean(img[:,:,2]) )
-
-        h_diff = img.shape[0] - h_n
-        w_diff = img.shape[1] - w_n
-
-        h_start = int(0.5 * h_diff)
-        w_start = int(0.5 * w_diff)
-
-        img_res = resize_image(img, h_n, w_n)
-        #label_res = resize_image(label, h_n, w_n)
-
-        img_scaled_padded = np.copy(img)
-
-        #label_scaled_padded = np.zeros(label.shape)
-
-        img_scaled_padded[:,:,0] = channel0_avg
-        img_scaled_padded[:,:,1] = channel1_avg
-        img_scaled_padded[:,:,2] = channel2_avg
-
-        img_scaled_padded[h_start:h_start+h_n, w_start:w_start+w_n,:] = img_res[:,:,:]
-        #label_scaled_padded[h_start:h_start+h_n, w_start:w_start+w_n,:] = label_res[:,:,:]
-
-        return img_scaled_padded#, label_scaled_padded
-
-    def do_prediction_new_concept_scatter_nd(
```

Comment:
I highly recommend against removing this function. It is still experimental, but can further speed up prediction immensely (because patch processing will be on GPU only instead of CPU-GPU back and forth).